src/pages/Signup.jsx

```diff
@@ -22,6 +22,7 @@ const SignupScreen = () => {
   const [username, setUsername] = useState("");
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [cpassword, sPassword] = useState("");
```

Comment:
cpassword is not readable, and sPassword is worse to read and understand. Try confirmPassword and setConfirmPassword.